router/src/infer.rs

```diff
@@ -138,7 +138,8 @@ impl Infer {
             }
         }
 
-        let adapter = Adapter::new(adapter_id.unwrap(), adapter_source.unwrap(), adapter_idx);
+        let predibase_api_token = request.parameters.predibase_api_token.clone();
```

Comment:
Can we get this from the bearer token in the header so the user dosn't need to put it in the request body?

Reply:
Actually, on second thought, that wouldn't work well for HF per my other comment, so I'm fine keeping this as a param.